### PR TITLE
Adjust Rich Text Editor Test to Remove Flackiness

### PR DESCRIPTION
**Pull request by SudharakaP**
Trying to fix the flacky test for the rich text editor; discussion at; https://opencollective.slack.com/archives/C0RMV6F8C/p1622102297046800

### CODE REVIEW

**Review by znarf — APPROVED** [review]
Looks good!

Let's wait on feedback from @Betree before merging.

**Review by Betree — APPROVED** [review]
Lgtm!

**Pull request merged**
